AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/participant-details/participant-details.component.html

```diff
@@ -5,41 +5,41 @@
       <img src="/assets/images/govuk-crest.png" height="40" width="40" class="govuk-header__logotype-crest"
         alt="HM Courts &amp; Tribunals Service crest" />
     </div>
-    <div class="govuk-grid-column-one-half vh-image-50" id="judge-name">
+    <div id="judge-name" class="govuk-grid-column-one-half vh-image-50">
       {{participant?.DisplayName}}
-      <div [attr.id]="'participant_role' + participant?.IndexInList" class="govuk-hint">
+      <div id="judge-hearing-role-name" class="govuk-hint">
         {{participant?.HearingRoleName}}
       </div>
     </div>
   </div>
 
   <div class="govuk-grid-column-full">
-    <div *ngIf="participant?.HearingRoleName !== 'Judge'">
+    <div id="participant-{{participant?.ParticipantId}}-name" *ngIf="participant?.HearingRoleName !== 'Judge'">
```

Comment:
should ParticipantId be in camel case (i.e. participantId)

Reply:
that's coming from the ParticipantDetailsModel TS class, so not sure.